vllm/v1/core/sched/scheduler.py

```diff
@@ -1283,16 +1281,15 @@ def _update_from_kv_xfer_finished(self,
 
         if self.connector is not None:
             self.connector.update_connector_output(kv_connector_output)
+            events = list(self.connector.take_events())
```

Comment:
Could check whether return value is truthy first to avoid creating an empty list (in case this isn't implemented).